lib/veri.ml

```diff
@@ -0,0 +1,207 @@
+open Core_kernel.Std
+open Bap.Std
+open Bap_traces.Std
+module Bap_value = Value
```

Comment:
I need .. what ? alias for `Value` module ?  Because I want to open `Result`, that contains one more `Value` module